packages/search/package.json

```
@@ -0,0 +1,64 @@
+{
+  "name": "navi-search",
+  "version": "0.2.0",
+  "description": "Search for navi",
```

```suggestion
  "description": "Search for navi",
  "private": true,
```

Make it private so that it builds but doesn't publish